com.microsoft.mrtk.core/Utilities/XRSubsystemHelpers.cs

```diff
@@ -151,6 +154,23 @@ public static XRDisplaySubsystem DisplaySubsystem
             }
         }
 
+        private static HandsAggregatorSubsystem handsAggregator = null;
+
+        /// <summary>
+        /// The currently loaded and running hands aggregator, if any.
+        /// </summary>
+        public static HandsAggregatorSubsystem HandsAggregator
+        {
+            get
+            {
+                if (handsAggregator == null || !handsAggregator.running)
```

Comment:
Out of curiosity, have you happened to do much profiling on this `.running` call? I'm hoping it's just a standard local property somewhere that doesn't have to go digging around for the state. Mostly asking because we do a lot of `if (XRSubsystemHelpers.HandsAggregator != null && XRSubsystemHelpers.HandsAggregator.TryDoSomething())`, which will effectively be null checking three times and checking `.running` twice. I don't expect that to be overbearingly expensive, but it could be worth profiling. Maybe the compiler optimizes it? Idk

Reply:
I'll do some profiling.

Reply:
Profiled, 1000 calls to `.running` registers 0.00ms in profiler